tests/test_forms.py

```diff
@@ -83,6 +84,44 @@ def test_data_url(self):
         assert widget.get_url() == '/foo/bar'
 
 
+class TestSelect2MixinSettings(TestCase):
```

Comment:
I noticed this.

I am happy to do this but:
a) I have had a lot of trouble (i.e. still can't) get the tests running with py.test on my dev box (is there something i'm missing?)
b) The tests rely on this [modify settings](https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.SimpleTestCase.settings)